### PR TITLE
feat(function): migrate to function

### PR DESCRIPTION
**Pull request by haarchri**
### Description of your changes
- migrate to function

Fixes #

I have:

- [x] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### CODE REVIEW

File: crossplane.yaml [hunk 1]
```diff
@@ -8,7 +8,11 @@ metadata:
     meta.crossplane.io/license: Apache-2.0
 spec:
   crossplane:
-    version: ">=v1.13.2-0"
+    version: ">=v1.14.1-0"
```
Comment (ytsarev) [inline crossplane.yaml:11]: Should we require higher(latest?) patchset here?

**Review by ytsarev — APPROVED** [review]
LGTM, one question on minimum required Crossplane version only

File: crossplane.yaml (hunk 1 above)
Comment (haarchri) [reply crossplane.yaml:11]: Good Catch

Comment (haarchri) [reply crossplane.yaml:11]: think its fine because we using `>= ` i enabled renovate for our configuration-* repos that we get these updates automatic

**Pull request merged**